repondeur/zam_repondeur/templates/lectures_add.html

```diff
@@ -29,12 +29,12 @@ <h2>Ajouter une lecture...</h2>
         </div>
         <div class="form-group">
             <label for="lecture">Lecture</label>
-            <select id="select-lecture" name="lecture" class="form-control" placeholder="Choisir une lecture...">
+            <select id="select-lecture" name="lecture" class="form-control" placeholder="Choisir une lecture..." autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false">
                 <option value="" selected="selected"></option>
             </select>
         </div>
         <div class="form-group">
-            <input type="submit" name="submit" value="Ajouter" class="btn btn-primary btn-lg">
+            <input type="submit" name="submit" value="Ajouter" class="btn btn-primary btn-lg" disabled>
```

Comment:
We probably want to have a transversal script to deal with/avoid submit button (double)submission

Reply:
Happy to refactor this into a more generic solution when we improve other forms!